examples/ZeroMQ/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ZeroMQ)
+
+hunter_add_package(ZeroMQ)
+
+find_package(PkgConfig)
+set(PKG_CONFIG_USE_CMAKE_PREFIX_PATH ON)
+pkg_search_module(ZeroMQ REQUIRED libzmq)
```

Comment:
It will not work for Windows, right? If yes then it's better to add `hunter_report_broken_package` to the `hunter.cmake` so the error reported before build started (and message will be more clearer). Like this https://github.com/ruslo/hunter/blob/c006432102cf21bf5171d38fd0a8fc7c4a3a183f/cmake/projects/BZip2/hunter.cmake#L18